x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/utils.ts

```diff
@@ -4,13 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { OutputRuleAlertRest } from '../../types';
-
-export const TEST_BOUNDARY = 'test_multipart_boundary';
+import { Readable } from 'stream';
 
-// Not parsable due to extra colon following `name` property - name::
-export const UNPARSABLE_LINE =
-  '{"name"::"Simple Rule Query","description":"Simple Rule Query","risk_score":1,"rule_id":"rule-1","severity":"high","type":"query","query":"user.name: root or user.name: admin"}';
+import { OutputRuleAlertRest } from '../../types';
+import { HapiReadableStream } from '../../rules/types';
```

Comment:
@yctercero These helpers all changed as a consequence of the refactor. We're no longer testing Hapi's ability to convert our request attachments to streams, but IMO that's a good thing.

Reply:
Yea, it looks much cleaner and you're right, now we're actually testing just the imports code written as opposed to Hapi's ability to convert.